packages/backend/src/jwt/types.ts

```diff
@@ -1,9 +1,9 @@
 export type JwtReturnType<R, E extends Error> =
   | {
       data: R;
-      error?: undefined;
+      errors?: undefined;
     }
   | {
       data?: undefined;
-      error: E;
+      errors: [E];
```

Comment:
with this type we make it explicit that the errors in this case will have a maximum of 1 item.